File: .py/Program/HDD.py
from Storage import Storage

class HDD (Storage):
    __RPM = 0

    def __init__ (self, brand = "", model = "", size = 0, RPM = 0):
        super ().__init__ (brand, model, size)
        self.__RPM = RPM

    def setRPM (self, RPM):
        self.__RPM = RPM

    def getRPM (self):
        return self.__RPM
    
    def info (self):
        super ().info ()
        print (f"HDD { self.__RPM }RPM")

File: .cpp/Program/HDD.cpp
#pragma once
#include <bits/stdc++.h>
#include "Storage.cpp"
using namespace std;

class HDD : public Storage {
    private:
        int RPM;

    public:
        HDD () { }

        HDD (string brand, string model, int size, int RPM) : Storage (brand, model, size) {
            this->RPM = RPM;
        }

        void setRPM (int RPM) { this->RPM = RPM; }

        int getRPM () { return RPM; }

        void info () override { Storage::info(); cout << " HDD " << RPM << "RPM" << endl; }

        ~HDD () { }
};